elasticsearch/src/main/java/com/yahoo/ycsb/db/ElasticSearchClient.java

```diff
@@ -90,13 +92,15 @@ public void init() throws DBException {
         .parseBoolean(props.getProperty("elasticsearch.remote", "false"));
     Boolean newdb =
         Boolean.parseBoolean(props.getProperty("elasticsearch.newdb", "false"));
-    Builder settings = settingsBuilder().put("node.local", "true")
+    Builder settings = Settings.settingsBuilder()
+        .put("node.local", "true")
```

Comment:
The default behavior for ES was set to point to local datastore. If we want to change that behavior, I'd put it in a separate commit.